llm-lora-finetuning/pipelines/train.py

```diff
@@ -0,0 +1,70 @@
+# Apache Software License 2.0
+#
+# Copyright (c) ZenML GmbH 2024. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from steps import configure, evaluate_model, finetune, prepare_data, promote
+from zenml import logging as zenml_logging
+from zenml import pipeline
+from zenml.config import DockerSettings
+
+zenml_logging.STEP_LOGS_STORAGE_MAX_MESSAGES = (
+    10000  # workaround for https://github.com/zenml-io/zenml/issues/2252
+)
+
+
+@pipeline(
+    settings={
+        "docker": DockerSettings(
+            parent_image="pytorch/pytorch:2.2.2-cuda11.8-cudnn8-runtime",
```

Comment:
I had to switch the image when switching from AWS to GCP as they require different CUDA versions in their VM. Maybe it makes sense to have that in the `config.yaml` as well for people to easily modify according to which version they need?

Reply:
Done